### ISSUE TITLE
HTML Markup field in forms are not styled / Preferred method for instructional text above form?

### ISSUE THREAD

**Issue by mark-bennett-uiowa** [uiowa/uiowa#3133]
# Problem
When creating a form for an information session to reproduce this page: [https://clas.uiowa.edu/socialwork/undergraduate-program/ba-programs/iowa-city-ba-program/information-sessions](https://clas.uiowa.edu/socialwork/undergraduate-program/ba-programs/iowa-city-ba-program/information-sessions), what is the preferred method moving forward in Drupal 8? Attach the form as a block in a basic page? Add a HTML Markup field above form so that both instructional text and form can be edited in the same window vs two? Third option?

Problem 1: HTML markup fields do not have a text format option, so styling does not carry over. The problem arises from the use of a HTML markup field in the form. The first screenshot shown below is what this HTML Markup field looks like live:
<img width="1319" alt="Screen Shot 2021-02-24 at 2 40 55 AM" src="https://user-images.githubusercontent.com/3067017/108973894-866ed500-764a-11eb-9f8d-b2477390e3f7.png">

This second screenshot is what it looks like when a Message field is used:
<img width="1330" alt="Screen Shot 2021-02-24 at 2 41 08 AM" src="https://user-images.githubusercontent.com/3067017/108973998-9f778600-764a-11eb-8f6d-ce6100957b62.png">

Should I be embedding this form as a block inside another page instead?

# Steps to reproduce
Create a basic form, and add a HTML Markup field to the form. No text formats are available, so no styling is available.

# Proposed solution
Allow Plain text and Filtered HTML text formats on the HTML Markup field?
Embed form as block in separate page?
Preferred method determined?

**Comment by joewhitsitt** [comment]
We can update these default webform settings to be more inline with the allowed minimal text format:

<img width="697" alt="Screen Shot 2021-02-24 at 9 29 02 AM" src="https://user-images.githubusercontent.com/4663676/109024001-e4b7aa00-7682-11eb-94e0-6bb8207d23dc.png">
<img width="374" alt="Screen Shot 2021-02-24 at 9 29 18 AM" src="https://user-images.githubusercontent.com/4663676/109024005-e5504080-7682-11eb-9411-b9faa7367383.png">

https://github.com/uiowa/uiowa/blob/main/config/default/filter.format.minimal.yml#L50

**Comment by mark-bennett-uiowa** [comment]
@joewhitsitt I followed your lead here; I unchecked Tidy HTML markup (to prevent it from stripping paragraph tags, I changed the Element text format to Filtered HTML, and tried turning on the Advanced HTML/Text element type option.

By changing the Element text format to Filtered HTML, it immediately fixed the issue. By using the Advanced HTML/Text, this also fixed it, but it raised two alerts and ends up being unnecessary if the standard HTML Markup field gets the job done. Unchecking Tidy HTML did nothing, so I left that checked.

TL;DR: By simply changing the Element text editor to Filtered HTML, it solved the problem. Closing issue. Thanks again Joe!!
[https://socialwork.prod.drupal.uiowa.edu/undergraduate/ba-social-work/information-sessions/iowa-city](https://socialwork.prod.drupal.uiowa.edu/undergraduate/ba-social-work/information-sessions/iowa-city)
![Screen Shot 2021-02-25 at 4 40 32 PM](https://user-images.githubusercontent.com/3067017/109229081-6f360180-7788-11eb-9e45-4602ff9ecebd.png)

BTW, here is the screenshot of the two alerts that were raised by enabling the Advanced HTML/Text field option:
![Screen Shot 2021-02-25 at 4 24 40 PM](https://user-images.githubusercontent.com/3067017/109229165-883eb280-7788-11eb-9cb1-b4e1e2ca9f32.png)
